plugins/woocommerce/src/Blocks/Shipping/ShippingController.php

```diff
@@ -232,6 +232,11 @@ public function register_settings() {
 	 * Hydrate client settings
 	 */
 	public function hydrate_client_settings() {
+		if ( ! wp_script_is( 'wc-shipping-method-pickup-location', 'enqueued' ) ) {
```

Comment:
Nitpick, but given how many times we reuse this string `wc-shipping-method-pickup-location` it might be good to set it in a constant.

Reply:
I converted this to a `private const`.